doc/ReferenceManual/GameModule.htm

```diff
@@ -5,592 +5,764 @@
+  <hr size="2" width="100%">
+  <h2>Module<br>
+  </h2>
+  <table border="0" cellpadding="2" cellspacing="2" width="100%">
+    <tbody>
+    <tr>
+    <td valign="top">The Module is the top component in the <a href="editor.htm">Editor</a> Configuration Window. Its properties are simply the
+    name of the game, a version number, and a description. These properties are each displayed separately their own
+    columns when the module is listed in the Module Manager that appears when Vassal starts up. The version number is
+    used to warn users who attempt to load save files created by a <i>later</i> version of a module into an <i>earlier
+    </i> one.<br>
+    <br>
+    All additions and modifications to a module are done through the <a href="editor.htm">Configuration Window</a>, also
+    frequently referred to as the <a href="editor.htm">Editor</a> window, which is a familiar file/folder type browser in
+    which each file/folder represents a module Component. The second illustration on the right shows the Editor's
+    configuration window for a "Brand New Module", with all of the Components that are added by default. Right-click on
+    the <i>[Module]</i> to get a context menu with options for the many types of <a href="#SubComponents">Components</a> you can add to a Vassal
+    module, including <a href="Map.htm">maps</a>, <a href="ChartWindow.htm">charts</a>, and <a href="PieceWindow.htm">
+    palettes</a> full of <a href="GamePiece.htm">game pieces</a>. See <a href="#SubComponents">below</a> for more
+    details.<br>
+    <br>
+    Actually <i><b>playing</b></i> the game takes place in the Module window (see example below). This window includes a
+    <a href="Toolbar.htm">Toolbar</a> that can be configured by the module designer, as well as a <a href="ChatLog.htm">Chat Log</a> to record
+    player messages and game moves. The game's main <a href="Map.htm">Map</a> will be displayed just below the Chat Log.
+    <br>
+    <br>
+    <div align="center">
+      <img src="images/ModuleWindow.png">
+    </div>
+    <div style="text-align:center; font-size:12px; padding-top:4px">
+      Module window during play, showing the <a href="Toolbar.htm">Toolbar</a>, a <a href="PieceWindow.htm">Game Piece Palette</a>, and the <a
+        href="ChatLog.htm">Chat Log</a>. The mapboard would appear below.
+    </div>
+    <br>
+    </td>
+    <td valign="top">
+    <img src="images/GameModule.png" alt="" title=""><br>
+    <div style="text-align:center; font-size:12px; padding-top:4px">
+      [Module] Properties
+    </div>
+    <br>
+    <br>
+    <br>
+    <img src="images/BrandNewModule.png"><br>
+    <div style="text-align:center; font-size:12px; padding-top:4px">
+      <a href="editor.htm">Editor</a> window with [Module] Component Highlighted
+    </div>
+    <br>
+    </td>
     </tr>
-
+    </tbody>
+  </table>
+  <hr size="2" width="100%">
+  <a href="Toolbar.htm"><h2>Toolbar</h2></a><br>
+
+  <img src="images/Toolbar.png"><br>
```

Comment:
Broken image.

Reply:
It's a new image, so if you haven't pulled down the full PR it won't find anything.

Reply:
fixed